Extension/gsnatives.cpp

```diff
@@ -88,6 +88,23 @@ static cell_t sm_IsLoaded(IPluginContext *pContext, const cell_t *params)
 	return IsSteamWorksLoaded() ? 1 : 0;
 }
 
+static cell_t sm_SetGameData(IPluginContext *pContext, const cell_t *params)
+{
+	ISteamGameServer *pServer = GetGSPointer();
+
+	if (pServer == NULL)
+	{
+		return 0;
+	}
+	
+	char *pData;
+	pContext->LocalToString(params[1], &pData);
+	
+	pServer->SetGameData(pData);
+	return 1;
+}
+
+
```

Comment:
Is this intended?

Reply:
Yes, I explicitly added an extra newline. My hopes were to trigger you.

Reply:
Can you please resolve this blocker?

Reply:
No can do. Sorry, brotato chip.